config/charts/knative-operator/templates/operator.yaml

```diff
@@ -6245,5 +6245,12 @@ spec:
           ports:
             - name: metrics
               containerPort: 9090
+          resources:
```

Comment:
Do these numbers come from any verification or such?

Reply:
Yes, I have verified this in my environment on a metrics basis and it works well with this value.

Reply:
If it's not a problem, I'll try to fix the unit test.

Reply:
@matzew Your opinion.
Does it seem safe to proceed with implementation?
Thx.